integration/src/test/scala/ammonite/integration/StandaloneTests.scala

```diff
@@ -63,6 +63,31 @@ object StandaloneTests extends TestSuite{
       println("\n-------------------------")
     }
 
+    'error_test{
+      val name = "Error_line_number_test.scala"
+      try {
```

Comment:
You should use [intercept](https://github.com/lihaoyi/utest#intercept) here to make sure that if the script *doesn't* throw, the test fails